action.yml

```
@@ -15,6 +15,9 @@ inputs:
     description: 'Optional comma separated list of coverage reports to send to Codacy'
     required: false
     default: ''
+  language:
+    description: 'Optionally force your project language'
```

Same suggestion as above here:

```suggestion
    description: 'Optionally force associating a language with your coverage report'
```